python/tvm/meta_schedule/integration.py

```diff
@@ -234,7 +234,7 @@ def extract_task_from_relay(
     if not isinstance(target, Target):
         target = Target(target)
 
-    with target, transform.PassContext(
+    with autotvm_silencer(), target, transform.PassContext(
```

Comment:
I'm still seeing:

```
WARNING:autotvm:One or more operators have not been tuned. Please tune your model for better performance. Use DEBUG logging level to see more details.
```

so probably we still need to silence AutoTVM to avoid confusion :-)